includes/system-drawing-common.md

```diff
@@ -0,0 +1,2 @@
+> [!NOTE]
+> In .NET 6 and later versions, the [System.Drawing.Common package](https://www.nuget.org/packages/System.Drawing.Common/), which includes this type, is only supported on Windows operating systems. Use of this type in cross-platform apps causes compile-time warnings and run-time exceptions. For more information, see [System.Drawing.Common only supported on Windows](/dotnet/core/compatibility/core-libraries/6.0/system-drawing-common-windows-only).
```

Comment:
should we instead use the aka.ms link I created? `aka.ms/systemdrawingnonwindows`

Reply:
For docs, we are advised against using aka.ms links. They are only recommended for links in the product that are much harder to update. Docs are easy to update, and it helps with our broken link checker not to have aka.ms links.